packages/communication-react/review/communication-react.api.md

```diff
@@ -851,6 +851,8 @@ export type CameraButtonSelector = (state: CallClientState, props: CallingBaseSe
 export interface CameraButtonStrings {
     cameraActionTurnedOffAnnouncement: string;
     cameraActionTurnedOnAnnouncement: string;
+    cameraButtonDisabledSplitAriaLabel?: string;
```

Comment:
Reading it now this will need to use `on`/`off` nomenclature to match the other props (and disabled for the camera button means unclickable instead of camera being "off".
In this interface it doesn't need to be prefixed with cameraButton either as these are the camera button strings:
```suggestion
    offSplitAriaLabel
    onSplitAriaLabel
```